addons/xivbar/xivbar.lua

```diff
@@ -0,0 +1,49 @@
+--[[
+        Copyright © 2017, SirEdeonX
+        All rights reserved.
+
+        Redistribution and use in source and binary forms, with or without
+        modification, are permitted provided that the following conditions are met:
+
+            * Redistributions of source code must retain the above copyright
+              notice, this list of conditions and the following disclaimer.
+            * Redistributions in binary form must reproduce the above copyright
+              notice, this list of conditions and the following disclaimer in the
+              documentation and/or other materials provided with the distribution.
+            * Neither the name of xivbar nor the
+              names of its contributors may be used to endorse or promote products
+              derived from this software without specific prior written permission.
+
+        THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS "AS IS" AND
+        ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE IMPLIED
+        WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE ARE
+        DISCLAIMED. IN NO EVENT SHALL SirEdeonX BE LIABLE FOR ANY
+        DIRECT, INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR CONSEQUENTIAL DAMAGES
+        (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES;
+        LOSS OF USE, DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND
+        ON ANY THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT
+        (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE OF THIS
+        SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+]]
+
+-- Addon Description
+_addon.name = 'XIV Bar'
+_addon.author = 'Edeon'
+_addon.version = '1.0'
+_addon.commands = {'xivbar'}
+_addon.language = 'english'
+
+-- Windower Libs
+config = require('config')
+texts  = require('libs/non-draggable-texts')
+images = require('libs/images')
```

Comment:
You've included a copy of windower's texts and images libraries in the directory of the addon, 
each with a minor change that switches the default setting for the dragging behavior to false.
It would be easier to require the unedited libraries and define your settings so that your
objects are not draggable.
This text object will not be draggable:
```
texts.new({
	flags = {
		draggable = false,
	},
})
```
I believe the images library has the same settings structure.